python/graphstorm/dataloading/dataloading.py

```diff
@@ -216,7 +278,74 @@ def fanout(self):
 BUILTIN_FAST_LP_LOCALUNIFORM_NEG_SAMPLER = 'fast_localuniform'
 BUILTIN_FAST_LP_LOCALJOINT_NEG_SAMPLER = 'fast_localjoint'
 
-class GSgnnLinkPredictionDataLoader():
+class GSgnnLinkPredictionDataLoaderBase():
+    """ The base class of link prediction dataloader.
+
+    Parameters
+    ----------
+    dataset: GSgnnEdgeData
+        The GraphStorm edge dataset
+    target_idx : dict of Tensors
```

Comment:
Could this target_idx be a Tensor, like in the `EdgeDataLoaderBase` and `NodeDataLoaderBase`?